### PR TITLE
appscale-tools: change PyPi URL to pythonhosted.org for `brew audit`

### PR DESCRIPTION
**Pull request by issyl0**
- As these are in an `unless OS.mac?` condition, they don't apply to the
  upstream Homebrew formula. These changes fix the following `brew
  audit` errors:
  ```
    * C: 319: col 7: https://pypi.python.org/packages/a5/a5/0830cfe34a4cfd0d1c3c8b614ede1edb2aaf999091ac8548dd19cb352e79/SecretStorage-2.3.1.tar.gz should be `https://files.pythonhosted.org/packages/a5/a5/0830cfe34a4cfd0d1c3c8b614ede1edb2aaf999091ac8548dd19cb352e79/SecretStorage-2.3.1.tar.gz`
  * C: 324: col 7: https://pypi.python.org/packages/3c/ec/a94f8cf7274ea60b5413df054f82a8980523efd712ec55a59e7c3357cf7c/pyparsing-2.2.0.tar.gz should be `https://files.pythonhosted.org/packages/3c/ec/a94f8cf7274ea60b5413df054f82a8980523efd712ec55a59e7c3357cf7c/pyparsing-2.2.0.tar.gz`
  ```

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] ~Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.~

-----

### CODE REVIEW

**Comment by iMichka** [comment]
```
c/_cffi_backend.c:15:17: fatal error: ffi.h: No such file or directory
compilation terminated.
error: command 'gcc-5' failed with exit status 1
```

**Comment by iMichka** [comment]
This often means the pkg-config is missing, to help the setup.py file to find `libffi`. But I am a little bit confused because `cffi` is also added as a resource.

**Comment by issyl0** [comment]
How strange. It builds fine for me locally, I don't know what to suggest.

**Comment by issyl0** [comment]
Just to let you know, I _can_ reproduce the build failure in the Linuxbrew Docker container[1]. That gives the same error as in CI. I'll continue looking for a fix to this.

If I couldn't reproduce it I was going to suggest running CI again to see if it's fixed itself. :smile: :computer: 

[1] - `linuxbrew/linuxbrew` which was last updated 3 months ago - is this still the right thing to run?

**Comment by issyl0** [comment]
@iMichka I found the fix, after going through some other formulae that had "libffi" as a dependency, the missing element for this was `depends_on "pkg-config" => :build`. This _should_ build fine in CI now, it builds fine in Docker for me. :crossed_fingers:

File: Formula/appscale-tools.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ class AppscaleTools < Formula
     sha256 "fb7096134ec7fd4172c3906190f123f21d8cf9c336729c07271f792e926997d3" => :el_capitan
   end
 
+  depends_on "pkg-config" => :build
```
Comment (iMichka) [inline Formula/appscale-tools.rb]: Please regroup this with the `depends_on "libffi"` line in a single `unless OS.mac? / end` block.

Comment (issyl0) [reply Formula/appscale-tools.rb]: Done!

**Comment by iMichka** [comment]
Thanks @issyl0 !

**Pull request closed without merging**